frontend/src/style/Header.css

```diff
@@ -1,28 +1,50 @@
+:root {
+  --grey: #808080;
+}
+
```

Comment:
This has been set up in `globals.css` already and is not needed here. Delete this declaration and the code will work the same